Source/EasyNetQ/FuturePublishConfiguration.cs

```diff
@@ -39,6 +47,7 @@ public FuturePublishConfiguration(string defaultTopic)
     public byte? Priority { get; private set; }
     public string Topic { get; private set; }
     public IDictionary<string, object?>? Headers { get; private set; }
+    public bool PropagateTraceContext { get; private set; }
```

Comment:
What do you think about enabling/disabling this on a global level?

I am highly unsure about having such a configuration here and in other similar places.

Reply:
> I am highly unsure about having such a configuration here and in other similar places.

So do I.